docs/04-reference/winglang-plugins.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Compiler Plugins
+id: compiler-plugins
+description: Wing Compiler Plugins Reference
+keywords: [Wing reference, Wing language, language, Wing language spec, Wing programming language, compiler plugins]
+---
+
+Wing compiler plugins are used to extend the Wing compiler. They can be used to add non-functional customizations to the compilation's resulting infrastructure. For example, you can use a compiler plugin to enforce a tagging convention on all resources in the resulting infrastructure.
```

Comment:
wrap to 100 width please ([rewrap] is amazing). Long lines make it harder to provide PR comments.

[rewrap]: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Reply:
Oh my! Rewrap is amazing, my markdown life is forever changed 😁